eslint.common.mjs

```diff
@@ -0,0 +1,47 @@
+import eslint from '@eslint/js';
+import tsEslint from 'typescript-eslint';
+import importPlugin from 'eslint-plugin-import';
+
+import {includeIgnoreFile} from "@eslint/compat";
+import path from "node:path";
+import {fileURLToPath} from "node:url";
```

Comment:
spacing around {

Reply:
This part of code is mostly removed to actually fetch `.gitignore` from root directories.